docs/use-cookie.md

```diff
@@ -0,0 +1,81 @@
+# useCookie
+
+Adds getters to easily read cookies and provides a simple API to set and clear cookies.
+
+Getters are then automatically defined by the mixin.
+- Getter name is a camelized conversion of the cookie name with the `Cookie` suffix. `user_id`  becomes `userIdCookie`
+
+**Options:**
+
+| Option| Description | Default value |
+|-----------------------|-------------|---------------------|
+| `expires` | Expire date of a cookie in days. Can be overriden on the options object on a per cookie basis |30|
+
+
+**Example**
+```js
+useCookie(this, { expires: 30 }) // Same as useCookie(this)
+```
+
+## Usage
+
+**Composing**
+
+```js
+import { Controller } from '@hotwired/stimulus'
+import { useCookie } from 'stimulus-use'
+
+export default class extends Controller {
+
+  connect() {
+    useCookie(this)
+    // Or pass an object to change the expires value (default 30 days):
+    // useCookie(this, { expires: 30 })
+
+    // Getters are initialized for all current cookies.
+    // For a cookie with name 'colorScheme' and value 'dark'
+    this.colorSchemeCookie // dark
+
+    // You can save new cookies with setCookie
+    // By passing two strings as name-value
+    this.setCookie("dessert", "cake")
```

Comment:
I'm wondering if it would be more natural from an API point of view, and also for symmetry, to use a more Stimulus-like way to set cookies. Something like:
```suggestion
    // You can save new cookies with the [name]Cookie setter
    this.desertCookie = "cake"
```

Or was there a reason against this?